docs/reference/ml/anomaly-detection/apis/find-file-structure.asciidoc

```diff
@@ -159,13 +163,13 @@ also specified.
 For structured file formats, if you specify this parameter, the field must exist
 within the file.
 
-If this parameter is not specified, the structure finder makes a decision about which
-field (if any) is the primary timestamp field. For structured file formats, it
-is not compulsory to have a timestamp in the file.
+If this parameter is not specified, the structure finder makes a decision about 
+which field (if any) is the primary timestamp field. For structured file 
+formats, it is not compulsory to have a timestamp in the file.
 --
 
 `timestamp_format`::
-  (string) Optional. The Java time format of the timestamp field in the file. +
+  (Optional, string) The Java time format of the timestamp field in the file.
 +
 --
 NOTE: Only a subset of Java time format letter groups are supported:
```

Comment:
This "NOTE" isn't showing up properly in the output and I think it's not necessary for this to be an admonition block.
```suggestion
Only a subset of Java time format letter groups are supported:
```